streaming/src/main/scala/org/apache/spark/streaming/ui/BatchPage.scala

```diff
@@ -37,10 +37,11 @@ private[ui] class BatchPage(parent: StreamingTab) extends WebUIPage("batch") {
   private def columns: Seq[Node] = {
     <th>Output Op Id</th>
       <th>Description</th>
-      <th>Output Op Duration</th>
+      <th>Output Op Duration {SparkUIUtils.tooltip("Time taken to handle all jobs of the batch",
```

Comment:
I don't think these tooltips are meaningful, not enough to add. For durations, the useful thing we can say is what the start and end time are exactly.

Reply:
itemIdAndFactors only directly depends on itemFactors and itemInBlocks.

Reply:
we should show the exact start time and end time of the job as tooltip? Or, do we need to define the start and end time here?

Reply:
No, I mean it should describe what the start and end time are. Like is start time when the task was submitted, starting running, etc.